inductiva/resources/machine_types.py

```diff
@@ -20,25 +20,22 @@ def get_available():
     return machine_types
 
 
-def list_available_machines():
+def list_available_machines(provider: str):
```

Comment:
can't we use get this listing directly from the BE?

```json
[
  {
    "machine_type": "n2d-highcpu-128",
    "num_cpus": 128,
    "ram_gb": 128,
    "price": 1.21344,
    "provider": "GCP",
    "spot": true,
    "region": "europe-west1"
  },
  ...
]
```

Reply:
Will do in another PR.